.github/workflows/reusable_push_pr.yaml

```diff
@@ -0,0 +1,116 @@
+name: Push/PR no fake prereleases
+
+on:
+  workflow_call:
+    inputs:
+      original_repo_name:
```

Comment:
NIT there should be a way to know it from the github metaadt of the caller, no? 
However, I fear that some repos could have it different from github 🤔